docs/src/install/gettingstarted.rst

```diff
@@ -0,0 +1,149 @@
+***************
+Getting Started
+***************
+
+Welcome! In this chapter, we give a quick overview of Oríon's main features and how it can help you
+streamline your machine learning workflow whether you are a researcher or engineer.
+
+Oríon is a black box function optimization library with a key focus on usability and integrability
+for its users. For example, as an machine learning engineer, you can integrate Oríon to your
+existing ML workflow to handle reliably the hyperparameter optimization and tuning phase. As a ML
+researcher, you can use Oríon to tune your models but also integrate your own algorithms to Oríon to
+serve as an efficient optimization engine and compare with other algorithms in the same context and
+conditions.
+
+Oríon is built to be non-intrusive, highly integrable, operate in parallel environments, and produce
+reproducible experiments. It takes only one line of code to start using it in your existing
+projects. It's natively asynchronous so it can perform efficiently on your laptop and in a computing
+farm with thousands of processors, without the need to configure a master and workers. Oríon
+supports the latest established hyperparameter algorithms out of the box, making it easy to switch
+between them or create benchmarks. It also supports a vast range of search spaces. Oríon uses a
+configuration agnostic approach where you can use any configuration file format you're comfortable
+with. The results are always stored in a database that can be either in-memory, local or remote.
+These results can be queried directly from the database itself or using convenient methods from
+Oríon's API. Additionally, the experiments are versioned -- think of it as a git for scientific
+experimentation -- enabling you to keep track of all your trials with their parameters. This
+guarantees that you can reproduce or trace back the steps in your work for free. Finally, a plugin
+system enables you to integrate and distribute your algorithms easily to other members of the
+community.
+
+Conversely, Oríon does not aim to be a machine learning framework or pipeline, or an automatic
+machine learning product. Oríon focuses essentially on black box optimization. However, we do
+encourage developers to integrate Oríon into that kind of systems as a component and we will do
+our best to help you if you're interested.
+
+Before continuing the overview, we assume that you have a basic understanding of machine learning
+concepts and that you already have installed Oríon on your machine and selected a database. If it's
+not the case please refer to our :doc:`installation instructions </install/core>` and :doc:`database
+setup </install/database>`.
+
+We also made a presentation_ if you prefer easy-to-read slides!
```

Comment:
> easy-to-read slides
Are they? :sweat_smile:

Reply:
They are! They have a good balance of information which is great :). Nonetheless, I replaced it with the bluejeans presentation. It's always nicer to have someone explain the slides for you!